<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowHead</class>
 <widget class="QDialog" name="MainWindowHead">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(31, 31, 31);</string>
  </property>
  <widget class="QToolButton" name="btnBrightScrs">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>110</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px white;
border-radius: 10px;
padding: 0 8px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/bright_scr.png</normaloff>:/new/prefix1/icons/bright_scr.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnLrf">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>320</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px white;
border-radius: 10px;
padding: 0 8px;
background: white;</string>
   </property>
   <property name="text">
    <string>LRF</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/lrf.png</normaloff>:/new/prefix1/icons/lrf.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOnOff">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>530</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px white;
border-radius: 10px;
padding: 0 8px;
background: white;</string>
   </property>
   <property name="text">
    <string>ON/
OFF</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/onoff.png</normaloff>:/new/prefix1/icons/onoff.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnPolarity">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>110</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px white;
border-radius: 10px;
padding: 0 8px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/palette.png</normaloff>:/new/prefix1/icons/palette.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDayIr">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>320</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>34</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px white;
border-radius: 10px;
padding: 0 8px;
background: white;</string>
   </property>
   <property name="text">
    <string>DAY</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/day.png</normaloff>:/new/prefix1/icons/day.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnZoom">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>530</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px white;
border-radius: 10px;
padding: 0 8px;
background: white;</string>
   </property>
   <property name="text">
    <string>1X</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/zoom1.png</normaloff>:/new/prefix1/icons/zoom1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QFrame" name="Label">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>50</y>
     <width>800</width>
     <height>640</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>790</width>
      <height>630</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
